CONTRIBUTING.md

```diff
@@ -139,15 +142,15 @@ Commits should be signed. You can read more about [Commit Signing][] here.
 We recommend a read on our [Collaborator Guide](COLLABORATOR_GUIDE.md#accepting-modifications) for in-depth details on how we accept contributions into this repository. The list below describes some of the merging and approval rules adopted in this repository.
 
 - Pull Requests must be open for at least 48 hours (Or 72 hours if the PR was authored on the weekend).
-  - Pull requests might be immediately merged if they contain critical bug fixes, short erratas (e.g. typos from previous PRs) or any critical change that is considered a "showstopper" for the operation of the website.
-    - These kind of PRs should only be done by existing collaborators that have write-access and/or signed-off by administrators/maintainers.
+  - Pull requests might be immediately merged if they contain critical bug fixes, short errata (e.g. typos from previous PRs) or any critical change that is considered a "showstopper" for the operation of the website.
+    - This kind of PRs should only be done by existing collaborators that have write-access and/or signed off by administrators/maintainers.
     - This rule cannot be used for updates on the COLLABORATOR_GUIDE, CONTRIBUTING guide, CODEOWNERS, GitHub Actions or any security-impacting file or document that changes the governing policies of this repository.
-  - Pull requests might be "fast-tracked", which means, they can be merged before the usual 48 hours notice if a "fast-track" label is added.
+  - Pull requests might be "fast-tracked", which means, they can be merged before the usual 48 hours' notice if a "fast-track" label is added.
     - The person that is fast-tracking the PR (adding the label) must also comment on the PR that they're requesting the PR to be fast-tracked
     - The comment must mention `@nodejs/website` and must have at least one 👍 (or any other sort of approval reaction) if the person fast-tracking the PR is the author of the PR.
-    - Fast-tracking is only allowed for small bug fixes, small feature changes, localisation changes, or other sort of non-critical/highly-impacting changes that are not covered by the previous rule that allows PRs to be merged immediately.
+    - Fast-tracking is only allowed for small bug fixes, small feature changes, localisation changes, or other sorts of non-critical/highly-impacting changes that are not covered by the previous rule that allows PRs to be merged immediately.
     - Fast-tracking cannot be used for updates on the COLLABORATOR_GUIDE, CONTRIBUTING guide, CODEOWNERS, GitHub Actions or any security-impacting file or document that changes the governing policies of this repository.
-- There must be no objections after a 48 hour period (Or 72 hours if the PR was authored on the weekend).
+- There must be no objections after a 48-hour period (Or 72 hours if the PR was authored on the weekend).
```

Comment:
Nit: just to make it same styling (same with the line 148)
```suggestion
- There must be no objections after a 48-hours period (Or 72-hours if the PR was authored on the weekend).
```
Or
```suggestion
- There must be no objections after a 48-hour period (Or 72-hour if the PR was authored on the weekend).
```

Reply:
I just checked again...
"48-hour period" is the correct way of writing it...
Same with "72 hours"